source/whats-new.txt

```diff
@@ -182,6 +183,34 @@ The {+driver-short+} v6.0 release includes the following features:
 To learn more about this release, see the
 `v6.0.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v6.0.0>`__.
 
+.. _version-5.9:
+
+What's New in 5.9
+-----------------
+
+.. important:: Upgrade from v5.8 to v5.9
+
+   This version includes a fix to a memory leak introduced in v5.7.
+   We recommend upgrading to v5.9.
```

Comment:
Suggestion:
I think a similar callout could help users for the v5.8 entry since the memory leak exists in v5.8 (according to this callout).